docs/metrics.rst

```diff
@@ -66,4 +66,5 @@ Timers
 Name                              Description
 ================================= =======================================
 dagrun.dependency-check.<dag_id>  Seconds taken to check DAG dependencies
+<dag_id>.<task_id>.duration       Seconds taken to finish a task
```

Comment:
I feel this should be prefixed with something to make the metrics un-ambigious. `dag.` (so `dag.mydag.my-task.duration = 300`)

Reply:
yeah that make sense.